src/devices/tuya.ts

```diff
@@ -6322,6 +6322,27 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_mgxy2d9f']),
```

Comment:
Reason  for using a fingerprint is that I know there's other models of this mention sensor, I'm just not sure what the manufacturer names are. 

EX, I know that this motion sensor the Meian SP02 is the same: https://www.aliexpress.us/item/3256804475730796.html